src/dialog.rs

```diff
@@ -15,9 +15,14 @@ use auto::DialogExt;
 use ResponseType;
 
 impl Dialog {
-    pub fn new_with_buttons<T: IsA<Window>>(title: Option<&str>, parent: Option<&T>,
-            flags: DialogFlags, buttons: &[(&str, ResponseType)]) -> Dialog {
+    pub fn new_with_buttons<'a, I: Into<Option<&'a str>>, T: IsA<Window>>(
+        title: I,
+        parent: Option<&T>,
```

Comment:
This don't changed on purpose?

Reply:
We didn't implement `Into` on `IsA` yet so not for now.